.github/workflows/build-libs.yml

```
@@ -30,17 +30,16 @@ jobs:
       - name: Install build dependencies
         run: |
           sudo apt-get update
-          sudo apt-get install build-essential git ocaml ocaml-nox ocaml-native-compilers camlp5 libgtk2.0 libgtksourceview2.0 liblablgtk-extras-ocaml-dev ocaml-findlib libnum-ocaml-dev emacs
+          sudo apt-get install coq
+          type coqc
+          coqc --version
 
       # Change into UniMath directory and build UniMath
       - name: Build UniMath and libraries TypeTheory, Largecatmodules, and SetHITs
         run: |
           pushd $GITHUB_WORKSPACE
-          make build-coq
           make -j2
           make install
```

It is possible that this command requires `sudo` now
```suggestion
          sudo make install
```

---

I'm working on it.  I'll use "sudo" to make the directory writable by all.

---

But now I'm puzzled that the makefile seems to recompile the *.vo needlessly.

---

> But now I'm puzzled that the makefile seems to recompile the *.vo needlessly.

What do you mean by that? At this point, no *.vo files exist yet, I believe.

---

I mean, I reproduced the problem at home.  The command "make install" built all the *.vo files and then failed trying to put them in "user-contrib".   Now I fixed that little problem and it's making all the *.vo files again.  Users won't like that.